src/main/java/core/basesyntax/service/impl/ProcessServiceImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ProcessService;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ProcessServiceImpl implements ProcessService {
+
+    private static final String REGEX = ",";
```

Comment:
Make it char instead of String

Reply:
The split method takes a string as a parameter